R/derive_vars_query.R

```diff
@@ -1,4 +1,7 @@
-#' Derive Query Variables
+#' Get Query Variables
+#'
+#' **Note:** This function exists to work with derive_vars_query, to execute
```

Comment:
Can we get a brief summary of the purpose of the function - similar to what you did in `get_flagged_records`.  

![image](https://github.com/pharmaverse/admiral/assets/10111024/a320ca50-5cc9-4af9-b699-af9096c7d00c)